src/v3/src/transformer/profile/transformEnrollProfile.ts

```diff
@@ -97,6 +97,13 @@ export const transformEnrollProfile: IdxStepTransformer = ({ transaction, formBa
       validate: (data: FormBag['data']) => {
         const newPw = data['credentials.passcode'] as string;
         const errorMessages: WidgetMessage[] = [];
+        if (!newPw && passwordElement.options.inputMeta.required) {
+          errorMessages.push({
+            class: 'ERROR',
+            message: loc('model.validation.field.blank', 'login'),
+            i18n: { key: 'model.validation.field.blank' },
+          });
+        }
```

Comment:
Since we are overriding the default validation function, we need to check if the field is required and trigger the empty field error message.